integration/buckets_test.go

```diff
@@ -208,6 +209,108 @@ func TestAddBucket(t *testing.T) {
 	}
 }
 
+func TestAddBucketLocking(t *testing.T) {
+	/*
+		This function is to test that locking can't be activated if versioning
+		is not enabled.
+		Then, locking will be activated because versioning is activated as well.
+	*/
+	assert := assert.New(t)
+
+	client := &http.Client{
+		Timeout: 2 * time.Second,
+	}
+
+	/*
+		This is invalid, versioning has to be true for locking to be true, but
+		test will see and make sure this is not allowed and that we get proper
+		error for this scenario.
+	*/
+	requestDataAdd := map[string]interface{}{
+		"name":       "test1",
+		"versioning": false,
+		"locking":    true,
+	}
+
+	requestDataJSON, _ := json.Marshal(requestDataAdd)
+
+	requestDataBody := bytes.NewReader(requestDataJSON)
+
+	// get list of buckets
```

Comment:
this code block creates a bucket so the comment can be updated accordingly.

```suggestion
	// create a object locking bucket without versioning flag
```

Reply:
The change has been performed.
Thank you for the suggestion @prakashsvmx